src/main/resources/log4j.properties

```diff
@@ -0,0 +1,8 @@
+# Set root logger level to INFO and its only appender to A1.
+log4j.rootLogger=INFO, A1
+log4j.appender.A1=org.apache.log4j.ConsoleAppender
+log4j.appender.A1.layout=org.apache.log4j.PatternLayout
+log4j.appender.A1.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
+
+log4j.logger.io..routerfs.*=INFO, A1
+
```

Comment:
This changes logging behaviour IIUC.  Isn't this something that the end-user should configure?

Reply:
You're right! It was supposed to be under the tests dir.
thanks